devices/profalux.js

```diff
@@ -0,0 +1,46 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const utils = require('../lib/utils');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const tzLocal = {
+    profalux_cover_state: {
+        key: ['state'],
+        convertSet: async (entity, key, value, meta) => {
+            switch (value) {
+            case 'OPEN':
+                await entity.command('genOnOff', 'on', {}, utils.getOptions(meta.mapped, entity));
+                break;
+            case 'CLOSE':
+                await entity.command('genOnOff', 'off', {}, utils.getOptions(meta.mapped, entity));
+                break;
+            case 'STOP':
+                await entity.command('genLevelCtrl', 'stop', {}, utils.getOptions(meta.mapped, entity));
+                break;
+            default:
+                throw new Error(`Value '${value}' is not a valid cover position (must be one of 'OPEN' or 'CLOSE' or 'STOP')`);
+            }
+        },
+    },
+};
+
+module.exports = [
+    {
+        fingerprint: [{type: 'Router', manufacturerID: 4368, endpoints: [{ID: 1, profileID: 260, deviceID: 512}]}],
```

Comment:
Why not also add the input/output cluster like https://github.com/Koenkk/zigbee2mqtt/issues/11206#issuecomment-1035290270?

Reply:
It did work well, but I tried to describe the fingerprint as close as other devices are when not using fingerprint (mainly using model and manufacturer name) so I modified the fingerprint in order to have only characteristics which seems releveant to me (manufId, used EP, profiletId and deviceId)
Do you think we'd better use the fingerprint you wrote?

Reply:
> Do you think we'd better use the fingerprint you wrote?

Yes, this reduces the chances of conflicts with other fingerprints

Reply:
Ok then
It seems theere's a duplication of the profalux file : 
https://github.com/Koenkk/zigbee2mqtt/issues/11362
and
https://github.com/Koenkk/zigbee-herdsman-converters/pull/3855

I'll let you merge the requests.

Note I also added support for SLV LED drivers : https://github.com/Koenkk/zigbee-herdsman-converters/pull/3863/commits/110f677b269492bb6447afeecbcc8a592f26898c
Not sure I did the GitHub part properly but the code is working so I let you merge this request as well.